homeassistant/components/vodafone_station/strings.json

```
@@ -29,5 +29,22 @@
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "external_ipv4": { "name": "WAN IPv4 address" },
+      "external_ipv6": { "name": "WAN IPv6 address" },
+      "external_ip_key": { "name": "WAN Internet Key address" },
```

Is "Internet Key" a brand name? If so, that's fine, if not:
```suggestion
      "external_ip_key": { "name": "WAN internet key address" },
```

---

Done